migrations/2023-08-11-073905_add_frm_config_in_mca/up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE "merchant_connector_account" ADD COLUMN frm_config jsonb[];
+ALTER TABLE "merchant_connector_account" DROP COLUMN IF EXISTS frm_configs;
```

Comment:
The running application asks for the column and the column is removed means that will break the current application which is running in environments.

So we can't drop a existing column which a application schema uses it.